api/BeforeUnloadEvent.json

```diff
@@ -30,10 +30,10 @@
             "version_added": "18"
           },
           "safari": {
-            "version_added": true
+            "version_added": "6.1"
```

Comment:
Basic support for the event goes back to https://trac.webkit.org/changeset/11259/webkit which was probably Safari 3.

The interface (this entry) was added here:
https://trac.webkit.org/changeset/155367/webkit
https://trac.webkit.org/browser/webkit/trunk/Source/WebCore/Configurations/Version.xcconfig?rev=155367

That would map to Safari 8, did https://mdn-bcd-collector.appspot.com/tests/api/BeforeUnloadEvent really test true in Safari 6.1?

Reply:
Yeah, this is testing `true` in Safari 6.1,  The test is pretty simple too, just `"BeforeUnloadEvent" in self`.

Reply:
That's interesting. But I trust testing over source code, maybe it lived in a patch outside of WebKit or something for a while.